markdown/3.x/en/core-result-dispatcher.md

```diff
@@ -107,3 +107,23 @@ registered callback.
 </script>
 <script defer src="pathToFFWebComponents/dist/bundle.js"></script>
 ```
+
+### `dispatchRaw(response, topics)`
+___
+Manually dispatches a **response** to the Web Components.
+This method is useful when there is a need to dispatch FACT-Finder response received in different way than by AJAX request, sent by Web Components.
+That will be the case especially when implementing [Server Side Rendering](/documentation/3.x/server-side-rendering).
+
+Argument **topics** is optional - skipping it causes the response will be dispatched to all default topics causing updates all the actively subscribing elements.
+With **topics** argument passed only a part of the response, related to the passed topic will be dispatched (e.g. using`suggest` will cause an update only on elements which subscribe to that topic - by default `ff-suggest-element`).
+**Note** It is possible to pass multiple topics in an array.
```

Comment:
Is the note intended to be inline? Using a new paragraph like the next note might be more of an eye-catcher.
Or a forced linebreak could be enough already.
```suggestion
**Note**: It is possible to pass multiple topics in an array.
```